cbpro/websocket_client.py

```diff
@@ -43,8 +44,8 @@ def _go():
 
         self.stop = False
         self.on_open()
-        self.thread = Thread(target=_go)
-        self.keepalive = Thread(target=self._keepalive)
+        self.thread = Thread(target=_go, name=thread_name)
+        self.keepalive = Thread(target=self._keepalive, name=keeaplive_thread_name)
```

Comment:
Should reference self.keepalive_thread_name. Also there is a typo in "keepalive"